### PR TITLE
GitHub Action for the linter

### PR DESCRIPTION
**Pull request by SilverLinings89**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Eslint
```
Comment (Cito) [inline .github/workflows/lint.yml:1]:
```suggestion
name: ESLint
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        node-version: [22]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install pnpm
```
Comment (Cito) [inline .github/workflows/lint.yml:13]: I would have expected that you first need to install Node.js and then pnpm, but it seems to work that way, too. So it's probably fine.

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: Eslint
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
```
Comment (Cito) [inline .github/workflows/lint.yml:8]: Would be easier to just specify the node version directly in the setup-node action and remove the matrix strategy. We don't need to run this on different node versions.

**Review by Cito — APPROVED** [review]
Looks good. Made some minor suggestions.

You also need to recreate the lock file.

File: .github/workflows/lint.yml (hunk 3 above)
Comment (SilverLinings89) [reply .github/workflows/lint.yml:8]: I used the recommended code from the pnpm documentation for this setup. I think I would keep it like that for now and update it if we notice any problems if that's OK

Comment (Cito) [reply .github/workflows/lint.yml:8]: It should no cause any problems, it's just an unnecessary complication. But I'm also fine with keeping it like that.

**Pull request merged**